app/main/views/users.py

```diff
@@ -296,3 +296,32 @@ def check_supplier_role(role, supplier_id):
         abort(400, "'supplier_id' is required for users with 'supplier' role")
     elif role != 'supplier' and supplier_id:
         abort(400, "'supplier_id' is only valid for users with 'supplier' role, not '{}'".format(role))
+
+
+@main.route("/buyers/briefs", methods=["GET"])
+def export_buyers_and_their_briefs():
+    buyers_with_briefs = db.session.query(User, Brief).\
+        outerjoin(BriefUser, User.id == BriefUser.user_id).\
+        outerjoin(Brief, BriefUser.brief_id == Brief.id).\
+        filter(User.role == 'buyer').\
+        order_by(User.name, Brief.id).\
+        all()
+
+    buyer_rows = []
+    last_email_address = ''
+    for buyer, brief in buyers_with_briefs:
+        if buyer.email_address == last_email_address:
+            last_email_address = buyer.email_address
+            buyer_rows[-1]['briefs'] = buyer_rows[-1]['briefs'] + ', {} - {}'.\
+                format(brief.data.get('title'), brief.status) if brief else None
+            continue
+        last_email_address = buyer.email_address
+        buyer_rows.append({
+            'buyer_name': buyer.name,
+            'buyer_email': buyer.email_address,
+            'buyer_phone': buyer.phone_number,
+            'buyer_created': buyer.created_at,
+            'briefs': '{} - {}'.format(brief.data.get('title'), brief.status) if brief else ''
+        })
```

Comment:
This is just a point of interest here because @allait wants a different approach taken, but what you're doing here can all be done _in_ the SQL using aggregations and `GROUP BY`. It's much more reliable, declarative and (usually much, much) faster to do it that way. https://www.postgresql.org/docs/9.5/static/tutorial-agg.html